backend/src/controllers/manager/adapterManger.ts

```diff
@@ -1,29 +1,25 @@
 import { NextFunction, Request, Response } from 'express';
 
-const applyMiddlewareManager = async (
+import { TLocalsManager } from 'interfaces/IResponse/IResponse';
+
+const adapterManger = async (

```

Comment:
If I get you right, here should be:
```suggestion
const adapterManager = async (
```
If it does, pls, change here and all the imports related to this module in the project.